waves-integration/src/main/scala/com/wavesplatform/dex/grpc/integration/clients/async/WavesBlockchainAsyncClient.scala

```diff
@@ -12,6 +17,26 @@ import monix.reactive.Observable
 object WavesBlockchainAsyncClient {
   type SpendableBalance        = Map[Asset, Long]
   type SpendableBalanceChanges = Map[Address, SpendableBalance]
+
+  private val briefWavesDescription =
+    Option(
+      BriefAssetDescription(
+        name = ByteStr("WAVES".getBytes(StandardCharsets.UTF_8)),
+        decimals = 8,
+        hasScript = false
+      ))
```

Comment:
Can we find better place for this code?
Looks a bit unnatural

Reply:
It is an internal optimization for Ops.assetDescription to avoid unnecessary creation of objects.

Reply:
If we place this constant into `BriefAssetDescription` companion object, optimization would not suffer

Reply:
Done